src/ResourceManager/SiteRecovery/Commands.SiteRecovery/help/Edit-AzureRmSiteRecoveryRecoveryPlan.md

```diff
@@ -100,7 +100,7 @@ Accept wildcard characters: False
 ```
 
 ### -Group
-Specifies a Site Recovery plan group.
+An instance of the ASRRecoveryPlanGroup which the plan belongs to.
```

Comment:
Its actually the other way around. Recovery Plan Groups are groupings under the recovery plan. There could be multiple Recovery plan groups in a recovery plan. I'd prefer we call it "Recovery Plan" and "Recovery Plan Group" as those are the nouns for these constructs. The class definition has ASR prefixed to distinguish this as an Azure Site Recovery (ASR) construct.